Gemfile

```diff
@@ -43,7 +43,6 @@ mingw << :x64_mingw if Bundler::Dsl::VALID_PLATFORMS.include?(:x64_mingw)
 platform(*mingw) do
   gem 'ffi', '~> 1.9.5', :require => false
   gem 'win32-dir', '~> 0.4.8', :require => false
-  gem 'win32-security', '~> 0.2.5', :require => false
```

Comment:
Removing this would affect a dev workflow using gems (and not the MSI) where a user has written custom facts, that depend on code in `win32-security`.

I feel like that scenario is pretty limited in scope, so this is probably OK.

Thoughts @ferventcoder / @joshcooper ?